first_lessons_dushanbe/tasks_from_lessons/tabl.py

```diff
@@ -4,12 +4,13 @@
 '''просто тренировочная таблица умножения на генераторы списков'''
 
 if __name__ == '__main__':
-    a = [[0] * 20 for i in range(20)]
-    for i in range(20):
-        for j in range(20):
+    b = 10
+    a = [[0]*b for i in range(b)]
+    for i in range(b):
+        for j in range(b):
             a[i][j] = (i + 1) * (j + 1)
```

Comment:
you can print and calculate values inside the same loop. Can you?

Reply:
done